docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,52 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve query plans by not generating redundant predicates. ({issue}`16520`)
+* Fix query failure when invoking the {func}`json_table` function. ({issue}`20122`)
```

Comment:
It think this is a table function .. and also I am not aware of any docs for it so dont use the func call. The rendering is adding () and that might be wrong.